djarg/tests/test_integration.py

```diff
@@ -206,6 +236,37 @@ def test_grant_staff_objects_view(client):
     assert 'bad_user2: Bad username' in content
 
 
+@pytest.mark.django_db
+def test_grant_staff_if_same_name_objects_view(client):
```

Comment:
Similarly, this test is based on the one immediately above it - tho more or less focused on the "same name" aspects of this action.